server/webserver.go

```diff
@@ -984,6 +984,32 @@ func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 
 	mux.HandleFunc("/requestTokens", func(w http.ResponseWriter, r *http.Request) {
 		if s.LivepeerNode.Eth != nil {
+
+			nextValidRequest, err := s.LivepeerNode.Eth.NextValidRequest(s.LivepeerNode.Eth.Account().Address)
+			if err != nil {
+				glog.Errorf("Unable to get the time for the next valid request from faucet: %v", err)
+				return
+			}
+			backend, err := s.LivepeerNode.Eth.Backend()
+			if err != nil {
+				glog.Errorf("Unable to get LivepeerEthClient backend: %v", err)
+				return
+			}
+			blkNum, err := s.LivepeerNode.Database.LastSeenBlock()
+			if err != nil {
+				glog.Errorf("Unable to get latest seen block number from database: %v", err)
+			}
+			blk, err := backend.BlockByNumber(context.Background(), blkNum)
```

Comment:
We can just pass in `nil` for the block number to fetch the latest block instead of reading the latest block number from the DB.

Reply:
fixed in eb24ec78